docs/content/users/install/performance.md

```diff
@@ -199,6 +201,13 @@ Mutagen can offer a big performance boost on macOS and Windows. It’s fast and
 
     ## NFS
 
+    !!!warning "NFS is deprecated"
+
+        Since users are happy with Mutagen, NFS is now deprecated and should
+        not be longer used anymore. This feature may be removed in an upcoming
+        release.
```

Comment:
Shorten and fix line wrapping:

```suggestion
    !!!warning "NFS is deprecated"

        NFS is now deprecated and should no longer be used. This feature may be removed in an upcoming release.
```

I don’t think justification is warranted in this warning—that’s something for a blog post or release notes maybe, worth linking to if it exists. Otherwise best to keep it direct.